Form-Controls/styles.css

```diff
@@ -0,0 +1,127 @@
+:root {
+    --white:#FFF;
+    --black:#000;
+    --db:#E6DACD;
+    --lb:#F4ECE6;
+    --rb:#0150FD;
+    --header-font: 'Poppins', sans-serif;
+    --gen-font: 'Quicksand', sans-serif;
```

Comment:
Suggest 'heading-font' and 'body-font' ? Other popular choices are "display" and "copy".